docs/TokenApi.md

```diff
@@ -5,6 +5,7 @@ All URIs are relative to *https://foo-api.esper.cloud/api*
 Method | HTTP request | Description
 ------------- | ------------- | -------------
 [**get_token_info**](TokenApi.md#get_token_info) | **GET** /v1/token-info/ | Token Information
+[**renew_token**](TokenApi.md#renew_token) | **GET** /v0/enterprise/{enterprise_id}/developerapp/{developerapp_id}/renew-token/ | Renew Token
```

Comment:
this is POST request

Reply:
Yeah. This autogenerated as I had given it as get in the esper-spec-api. I've changed that there and raised a PR for it.